### PR TITLE
Batch buildVariablesTree calls when initiating hover

### PR DESCRIPTION
**Pull request by CoderDake**
This change batches the buildVariablesTree calls when hovering. This fires them all off as closely as possible to each other so that they can take advantage of performance optimizations on evals that come in at close to the same time.

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/inspector/inspector_service.dart [hunk 1]
```diff
@@ -1200,8 +1151,9 @@ class ObjectGroup extends ObjectGroupBase {
 
   Future<RemoteDiagnosticsNode?> getRootWidget() {
     return parseDiagnosticsNodeDaemon(
-      invokeServiceExtensionMethod(
-        RegistrableServiceExtension.getRootWidgetSummaryTreeWithPreviews,
+      invokeServiceMethodDaemonParams(
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/inspector/inspector_service.dart:1154]: all of the changes to use `WidgetInspectorServiceExtensions` are from https://github.com/flutter/devtools/pull/4633, so these should be removed from this PR

File: packages/devtools_app/lib/src/screens/inspector/inspector_service.dart [hunk 2]
```diff
@@ -1158,35 +1138,6 @@ class ObjectGroup extends ObjectGroupBase {
   @override
   final InspectorService inspectorService;
 
-  Future<Object?> invokeServiceExtensionMethod(
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/inspector/inspector_service.dart:1161]: revert this change - part of https://github.com/flutter/devtools/pull/4633

File: packages/devtools_app/lib/src/screens/inspector/inspector_service.dart [hunk 3]
```diff
@@ -21,30 +21,10 @@ import '../../shared/eval_on_dart_library.dart';
 import '../../shared/globals.dart';
 import '../debugger/debugger_model.dart';
 import 'diagnostics_node.dart';
-import 'inspector_service_polyfill.dart';
 import 'primitives/inspector_common.dart';
 
 const inspectorLibraryUri = 'package:flutter/src/widgets/widget_inspector.dart';
 
-class RegistrableServiceExtension {
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/inspector/inspector_service.dart:29]: revert this. part of https://github.com/flutter/devtools/pull/4633

**Review by kenzieschmoll — APPROVED** [review]

**Comment by kenzieschmoll** [comment]
Please add a more descriptive PR description as this will be used to generate the changelog. thanks!

**Review by annagrin — APPROVED** [review]
LGTM!

File: packages/devtools_app/lib/src/screens/inspector/inspector_service.dart [hunk 4]
```diff
@@ -57,14 +57,6 @@ abstract class InspectorServiceBase extends DisposableController
...
           serviceManager.service!,
-          // TODO(jacobr): evaluate whether oneRequestAtATime is really required.
-          // The out of order request issues seen may have been isolated to Java
-          // where requests could truly be out of order due to multiple threads.
-          // It appears that enforcing in-order requests has significant negative
-          // consequences that out-weigh the benefits of being able to cancel
-          // requests from object groups that have been disposed before the requests
-          // were issued.
-          oneRequestAtATime: true,
           isolate: serviceManager.isolateManager.mainIsolate,
```
Comment (jacob314) [inline packages/devtools_app/lib/src/screens/inspector/inspector_service.dart:60]: we will want to make sure that the # of exceptions from the inspector does not go up after this release goes public. oneRequestAtATime. Removing this should help performance but may result in a lot more bugs along the lines of "inspector object id already disposed"
